docs/pages/inboxes/content-types/transactions.md

```diff
@@ -0,0 +1,90 @@
+# Support onchain transactions in your app built with XMTP
+
+This package provides an XMTP content type to support sending transactions to a wallet for execution. 
+
+:::tip[Open for feedback]
+
+You are welcome to provide feedback on this implementation by commenting on [XIP-59: Trigger on-chain calls via wallet_sendCalls](https://community.xmtp.org/t/xip-59-trigger-on-chain-calls-via-wallet-sendcalls/889).
+
+:::
+
+## Install the package
+
+```bash
+# npm
+npm i @xmtp/content-type-wallet-send-calls
+
+# yarn
+yarn add @xmtp/content-type-wallet-send-calls
+
+# pnpm
+pnpm i @xmtp/content-type-wallet-send-calls
+```
+
+## Create a transaction request
+
+With XMTP, a transaction request is represented using `wallet_sendCalls` with additional metadata for display:
+
+```tsx [Browser]
+const walletSendCalls: WalletSendCallsParams = {
+  version: "1.0",
+  from: "0x123...abc",
+  chainId: "0x2105",
+  calls: [
+    {
+      to: "0x456...def",
+      value: "0x5AF3107A4000",
+      metadata: {
+        description: "Send 0.0001 ETH on base to 0x456...def",
+        transactionType: "transfer",
+        currency: "ETH",
+        amount: 100000000000000,
+        decimals: 18,
+        toAddress: "0x456...def",
+      },
+    },
+    {
+      to: "0x789...cba",
+      data: "0xdead...beef",
+      metadata: {
+        description: "Lend 10 USDC on base with Morpho @ 8.5% APY",
+        transactionType: "lend",
+        currency: "USDC",
+        amount: 10000000,
+        decimals: 6,
+        platform: "morpho",
+        apy: "8.5",
+      },
+    },
+  ],
+};
+```
+
+## Send a transaction request
+
+Once you have a transaction reference, you can send it as part of your conversation:
+
+```tsx [Browser]
+await conversation.messages.send(walletSendCalls, {
+  contentType: ContentTypeWalletSendCalls,
+});
+Receive a transaction request
+To receive and process a transaction request:
```

Comment:
are these 2 lines meant to be in the code block? if so, they should be commented.